### ISSUE TITLE
Unexpected value of covariance matrix

### ISSUE THREAD

**Issue by Akella17** [cornellius-gp/gpytorch#856]
@jacobrgardner @gpleiss  The covariance matrix of a linear kernel is not coming out to be the same as (x.x<sup>T</sup> + &sigma;<sup>2</sup>*I*). Is there any form of normalization happening in the background?
```Python
print(model(train_x).covariance_matrix.sum())
print(Mvp(torch.ones(length).cuda()).sum()) # computes (x.x^T + \sigma^2.I).sum()
```
```
tensor(11560.1421, device='cuda:0')
tensor(1.0454e+12, device='cuda:0')
```

Please find the [example iPython notebook](https://drive.google.com/file/d/1Dg1bEb3_5g9Rf-BIBT71kyj4Q2aMy9p0/view?usp=sharing) that demonstrates this issue.
